src/julia/tests/test_juliaoptions.py

```diff
@@ -11,7 +11,7 @@
     (dict(compiled_modules="no"), ["--compiled-modules", "no"]),
     (dict(depwarn="error"), ["--depwarn", "error"]),
     (dict(sysimage="PATH"), ["--sysimage", "PATH"]),
-    (dict(bindir="PATH"), ["--home", "PATH"]),
```

Comment:
What happened to `bindir`?

Reply:
Other than this merge when ready.

Reply:
Is bindir an option in Julia anymore? I don't see it in the `--help` menu.

Reply:
https://github.com/JuliaLang/julia/blob/master/src/jloptions.c

Reply:
Not sure, I can't even use it locally:

```bash
julia --bindir=$PATH
# ERROR: unknown option `--bindir
```

Reply:
I think it's replaced by `--home`?

Reply:
Oh yeah, there it is:

```python
    def cli_argument_name(self):
        name = {"bindir": "home"}.get(self.name, self.name)
```

So this should not be removed @dpinol

Reply:
I readded it.